modin/pandas/groupby.py

```diff
@@ -278,12 +278,22 @@ def value_counts(
             )
         )
 
-    def mean(self, numeric_only=None):
+    def mean(self, numeric_only=no_default, engine="cython", engine_kwargs=None):
+        if engine not in ("cython", None) and engine_kwargs is not None:
```

Comment:
`cython` is the default value that pandas converts `None` value to. For example, https://pandas.pydata.org/pandas-docs/version/1.5.3/reference/api/pandas.core.groupby.GroupBy.std.html?highlight=std#pandas.core.groupby.GroupBy.std